exercises/hamming/tests/hamming.rs

```diff
@@ -32,11 +32,13 @@ fn test_larger_distance() {
 #[test]
 #[ignore]
 fn test_first_string_is_longer() {
-    assert_eq!(dna::hamming_distance("AAA", "AA"), Result::Err("inputs of different length"));
+    assert_eq!(dna::hamming_distance("AAA", "AA"),
+               Result::Err("inputs of different length"));
```

Comment:
not related to this PR, so change is not needed here, but I wonder if this should just be `is_err` instead, leave the message flexible. advantages to both